docs/src/_includes/layouts/doc.html

```diff
@@ -35,9 +37,20 @@ <h2 id="further-reading">Further Reading</h2>
         {% set all_content = [all_content, further_reading_content] | join %}
     {% endif %}
 
+    {% set rule_meta = rules_meta[title] %}
+
     <div class="docs-content">
         <main id="main" tabindex="-1" class="docs-main">
             <h1>{{ title }}</h1>
+            {% if rule_meta %}
+                <p>{{ rule_meta.docs.description | markdown | safe }}</p>
+                {{ ruleCategories({
+                recommended: rule_meta.docs.recommended,
+                fixable: rule_meta.fixable,
```

Comment:
The fixable blurb isn't showing on e.g. [`arrow-body-style`](https://deploy-preview-15902--docs-eslint.netlify.app/rules/arrow-body-style/), perhaps because the `ruleCategories` macro [expects this to be a boolean](https://github.com/eslint/eslint/blob/901ce0f1e32ea1e9e10ce4d8b37c0d750007a3c5/docs/src/_includes/components/rule-categories.macro.html#L12), but `rule_meta.fixable` will be either `"code"` or `"whitespace"`.

Reply:
Oops, good call.